force-app/main/default/lwc/staffAvailabilityItem/staffAvailabilityItem.js

```diff
@@ -35,13 +35,14 @@ export default class StaffAvailabilityItem extends LightningElement {
 		notifyViaAlerts: false // Use alerts instead of toast to notify user
 	};
 
-	renderedCallback() {
-
+	@api
+	get assignmentRecord() {
+		return '';
 	}
 
-	connectedCallback() {
-		this.record = {...this.assignment, sobjectType: this.namespace+'Assignment__c'};
-		this.statusValue = getFieldValue(this.record, 'Available__c', this.namespace);
+	set assignmentRecord(value) {
```

Comment:
this one binds the markup attributre? this is sooo cool

Reply:
yep, and make sure it changes the data both ways parent / child lwc so it rerenders when the values changes.